hack/check.sh

```diff
@@ -1,28 +0,0 @@
-#!/bin/bash
-
-set -e
-
-GOLANGCI_LINT_CONFIG_FILE=""
-
-if [ "$1" == "--fix" ]; then
-  opt="--fix"
-  shift
-fi
-
-
-for arg in "$@"; do
-  case $arg in
-    --golangci-lint-config=*)
-    GOLANGCI_LINT_CONFIG_FILE="-c ${arg#*=}"
-    shift
-    ;;
-  esac
-done
-
-echo "> Check"
-
-echo "Executing golangci-lint"
-echo "  golangci-lint run $GOLANGCI_LINT_CONFIG_FILE $opt --timeout 10m $@"
-golangci-lint run $GOLANGCI_LINT_CONFIG_FILE $opt --timeout 10m $@
```

Comment:
I don't know, why there was a timeout introduced in the first place... but now it's not set. Are we sure, it's ok to run without it?

Reply:
YOu're right, I definitely need to put that timeout back. It's now 5m now.